geometry_msgs/Pose2D pose
int8 length
int8 width
string color